src/core/TrainController.cpp

```diff
@@ -276,6 +276,9 @@ namespace godot {
     double TrainController::get_nominal_battery_voltage() const {
         return nominal_battery_voltage;
     }
+    void TrainController::set_brake_level(const double p_brake_level) {
+        //@TODO: Implement
```

Comment:
w hpp widze set_brake_level, ale IMO to tam jest błąd. jeśli korygować to wywalić z naglowka

Reply:
W hpp bylo wiec dodalem to do cpp, wywale z obu